docs/docs/gatsby-repl.md

```diff
@@ -1,9 +1,133 @@
 ---
 title: Gatsby REPL
-issue: https://github.com/gatsbyjs/gatsby/issues/7484
 ---
 
-This is a stub. Help our community expand it.
+The Gatsby REPL ("read-eval-print loop") is available via the Gatsby CLI by running the command `gatsby repl`. This gives you access to an interactive REPL shell within the context of your Gatsby environment. It can be used to retrived general data and programatically interact with it. If you have an idea for a new command, feel free to submit a PR for it!
```

Comment:
```suggestion
The Gatsby REPL ("read-eval-print loop") is available via the Gatsby CLI by running the command `gatsby repl`. This gives you access to an interactive REPL shell within the context of your Gatsby environment. It can be used to retrieve general data and programmatically interact with it. If you have an idea for a new command, feel free to submit a PR for it!
```